openhands/runtime/base.py

```diff
@@ -225,6 +225,13 @@ async def _handle_action(self, event: Action) -> None:
                         export_cmd = CmdRunAction(
                             f"export GITHUB_TOKEN='{token.get_secret_value()}'"
                         )
+
+                        self.event_stream.set_secrets(
+                            {
+                                'github_token': token.get_secret_value(),
+                            }
+                        )
```

Comment:
I think we need a `set_secret` function that just overrides this one var--otherwise we'll drop any other secrets that are there

Reply:
Thanks for the catch! Updated this accordingly

Reply:
Sorry to be nitpicky but I'd expect a setter to just set. Maybe we need an `update_secrets` in addition to `set_secrets`. Or maybe we just rename this method entirely since init sets it to `{}`